examples/hello_world_plugin/README.md

```diff
@@ -0,0 +1,43 @@
+# Engine plugin
+
+This example shows how to create a plugin that integrates a rendering engine with
+Ignition Rendering.
```

Comment:
Can you add a sentence that says this demo also shows how to load this plugin with Ignition Gazebo?

Reply:
0dd768276f5742bd222ca33340f1035a413542aa